src/index.js

```diff
@@ -1,24 +1,71 @@
 /* eslint-disable no-unused-vars */
```

Comment:
- [ ] My comment on linters applies here as well. Let's remove the unused var on our code.

Reply:
Thank you so much for your keen observation. I have managed to solve the concerns in my code that you requested and everything works fine.